osquery/logger/plugins/aws_util.h

```diff
@@ -0,0 +1,121 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <memory>
+
+#include <aws/core/Region.h>
+#include <aws/core/auth/AWSCredentialsProvider.h>
+#include <aws/core/auth/AWSCredentialsProviderChain.h>
+#include <aws/core/client/ClientConfiguration.h>
+#include <aws/core/http/HttpClient.h>
+#include <aws/core/http/HttpClientFactory.h>
+#include <aws/core/http/standard/StandardHttpResponse.h>
+
+#include <boost/property_tree/ini_parser.hpp>
```

Comment:
I don't think you need this here.

Also, can we convert the INIs into JSON? Eventually I'd like to drop boost's property tree APIs. If the only dependency is JSON parsing it'll be much easier.